ros/angel_utils/scripts/bag_extractor.py

```diff
@@ -332,7 +420,9 @@ def handle_image_msg(self, msg: Image) -> None:
         # Save image to disk
         timestamp_str = f"{msg.header.stamp.sec:011d}_{msg.header.stamp.nanosec:09d}"
         file_name = (
-            f"{self.image_folder}frame_{self.num_image_msgs:05d}_" + timestamp_str + ".png"
+            f"{self.image_folder}frame_{self.num_image_msgs:05d}_"
+            + timestamp_str
+            + ".png"
```

Comment:
Why not include the `timestamp_str` in the format string?
```suggestion
            f"{self.image_folder}frame_{self.num_image_msgs:05d}_{timestamp_str}.png"
```